vlib/db/pg/README.md

```diff
@@ -51,3 +51,13 @@ Read this section to learn how to install and connect to PostgreSQL
 [*Windows*](https://www.postgresqltutorial.com/install-postgresql);
 [*Linux*](https://www.postgresqltutorial.com/postgresql-getting-started/install-postgresql-linux);
 [*macOS*](https://www.postgresqltutorial.com/postgresql-getting-started/install-postgresql-macos).
+
+## Using Parameterized Queries
+
+To use parameterized queries (exec_param, exec_param2, exec_param_many) in V code the syntax `($1) ($2)...($n)` must be used, where the number following the $ specifies which parameter from the argument array will be filled in. 
```

Comment:
Is the following maybe a bit easier to follow? Splitting up the sentences and changing the wording a little:


Parameterized queries (`exec_param`, `exec_param2`, `exec_param_many`) in V require the use of the following syntax: `($1) ($2)...($n)`. The number following the `$` specifies which parameter from the argument array to use.

Reply:
Perfect. I like that better